src/manifest.json

```diff
@@ -10,31 +10,21 @@
     "128": "icons/icon128.png"
   },
   "permissions": [
+    "tabs",
+    "storage",
     "https://github.com/*",
     "https://gist.github.com/*"
   ],
-  "content_scripts": [
```

Comment:
Updated manifest to remove the original content scripts injection. This is to support the option to disable github/gist injection independently through the options page. The default for this functionality is to inject the scripts automatically.

The injection would now be handled by the background script.